elementary/monitor/cli.py

```diff
@@ -205,6 +206,12 @@ def report(ctx, days_back, config_dir, profiles_dir, update_dbt_package, profile
     default=None,
     help="The slack channel which all alerts will be sent to.",
 )
+@click.option(
```

Comment:
Its a bit confusing to pass a timezone to the send_report command when it doesn't affect anything.

As long as it has no use, let's not add it to this command.